src/Arc.cpp

```diff
@@ -1,5 +1,5 @@
-#include "roboteam_utils/Arc.h"
-#include "roboteam_utils/Math.h"
+#include "../include/roboteam_utils/Arc.h"
```

Comment:
The old style is better.This can be fixed in the CMake by properly using include directories. Might have happened automatically when you refactored names?